admin/files/online-prezence/online-prezence-potvrzovaci-modal.js

```diff
@@ -0,0 +1,34 @@
+/* logika k situaci, že není zaškrtnutý ani jeden účastník k aktivitě  */
+function zadnyUcastnikNeniPotvrzen() {
+  const checkboxy = document.querySelectorAll(`#ucastniciAktivity${posledneKliknutaAktivitaId} input.dorazil[checked]`)
+  return checkboxy.length === 0
+}
+
+function potvrdModal() {
+  if (zadnyUcastnikNeniPotvrzen()) {
+    if (!document.getElementById('potvrzujiBezUcastniku').checked) {
+      document.getElementById('labelPotvrzujiBezUcastniku').classList.add('fw-bold')
+      return
+    }
+  }
+  document.getElementById('online-prezence').dispatchEvent(
+    new CustomEvent('uzavritAktivitu', {detail: posledneKliknutaAktivitaId}),
+  )
+}
+
+document.addEventListener('DOMContentLoaded', function () {
+  Array.from(document.getElementsByClassName('tlacitko-uzavrit-aktivitu')).forEach(el => {
+    el.addEventListener('click', function () {
+      if (zadnyUcastnikNeniPotvrzen()) {
+        // žádné vybrané aktivity
+        document.getElementById('wrapperPotvrzujiBezUcastniku').classList.remove('d-none')
+        return
+      }
+      document.getElementById('wrapperPotvrzujiBezUcastniku').classList.add('d-none')
+    })
+  })
+
+  $('#modalOpravduUzavrit').on('hidden.bs.modal', function () {
+    document.getElementById('labelPotvrzujiBezUcastniku').classList.remove('fw-bold')
```

Comment:
Nově když se modal zavře, tak se červený nápis zase odtuční - to aby se přiště zase mohl zvýraznit a upozornit na sebe.